tests/rotatingMHD/vector_tools_02.mpirun=2.output

```diff
@@ -0,0 +1,16 @@
+Vector difference: 3.37113e-15
+L2 error: 3.81071e-16
+H1 error: 3.68109e-15
+Linfty error: 6.66134e-16
+Vector difference: 2.02131e-14
+L2 error: 8.71249e-16
+H1 error: 7.50576e-15
+Linfty error: 1.55431e-15
+Vector difference: 1.48952e-15
+L2 error: 1.57316e-16
+H1 error: 2.19841e-15
+Linfty error: 4.44089e-16
+Vector difference: 1.167e-14
+L2 error: 5.03016e-16
+H1 error: 4.33345e-15
+Linfty error: 1.55431e-15
```

Comment:
Won't this output depend on the machine precision of the computer where the test is to be performed?

Reply:
Yes, I will add `(error < 1e-12)` instead.